<launch>

    <!-- Run the map server -->
    <node name="map_server" pkg="map_server" type="map_server" args="$(find map_provider)/maps/map-hector.yaml"/>

</launch>